docgen/middlewares.js

```diff
@@ -1,22 +1,25 @@
-import layouts from 'metalsmith-layouts';
 import headings from 'metalsmith-headings';
-import navigation from 'metalsmith-navigation';
+import layouts from 'metalsmith-layouts';
 import markdown from 'metalsmith-markdown';
+import msWebpack from 'ms-webpack';
+import navigation from 'metalsmith-navigation';
+import sass from 'metalsmith-sass';
 
-import inlineProps from './plugins/inlineProps/index.js';
-import renderer from './mdRenderer.js';
-import source from './plugins/source.js';
-import onlyChanged from './plugins/onlyChanged.js';
 import assets from './plugins/assets.js';
-import ignore from './plugins/ignore.js';
 import helpers from './plugins/helpers.js';
+import ignore from './plugins/ignore.js';
```

Comment:
eslint-plugin-import is pretty advanced (we already use some rules), and maybe they have automatic fixable rules already :D https://github.com/benmosher/eslint-plugin-import

For now we are good, was just exploring the PR